### DISCUSSION TITLE
Can I use Spectrum outside of Adobe products?

### DISCUSSION THREAD

**Discussion by SavoInfostream** [adobe/spectrum-web-components#1954] [Q&A] [answered]
Hello,

Is Adobe Spectrum free to use for any project or is it limited to Adobe related products?

Thanks.

**Comment by Westbrook** [comment, marked as answer]
Absolutely. Checkout GitHub's cheatsheet on our licence terms:

[![GitHub license cheatsheet](https://user-images.githubusercontent.com/1156657/143771730-b66d3205-e3cc-461f-8a35-5bba669646c6.png)](https://github.com/adobe/spectrum-web-components/blob/main/LICENSE)

As you can see, Spectrum Web Components is a fully free and open source project for you to use whether you work is related to Adobe, or not:

https://github.com/adobe/spectrum-web-components/blob/d362050c2fd2c8b105bb32929923cfe69ddbb1b3/LICENSE#L66-L87
